opencdc/toproto.go

```diff
@@ -0,0 +1,110 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opencdc
+
+import (
+	"fmt"
+
+	opencdcv1 "github.com/conduitio/conduit-commons/proto/opencdc/v1"
```

Comment:
the package name is already opencdcv1, this is just to make it explicit?

Reply:
Yes, I think `goimports` added this, I never touch imports manually. It's probably because if you look at the import path it's not obvious that the package name is `opencdcv1`, this makes it explicit.